### ISSUE TITLE
Default sounds not re-enabling after turning off custom sounds

### ISSUE THREAD

**Issue by tposney** [tposney/midi-qol#148]
In GitLab by @jdonley18 on Nov 24, 2020, 20:06

I have a world where I used to use custom sounds through Midi-qol.  When I uncheck the box "enable midi-qol custom sounds", I'm not getting my default dice sound back.  If I uncheck "roll automation support" in the main options panel, the sounds return, so I feel confident the issue is with midi-qol.  Disabling midi entirely also brings the sounds back. I have other worlds where I use midi but never enabled the custom sounds and I get a dice sound in those worlds even with "roll automation support" checked.

**Comment by tposney** [comment]
Thanks for the report. I'll have a look. I can imagine that disabling the sounds checkbox might require a refresh, but it should be fine after that. Can you confirm that disabling and doing a reload/refersh does not bring the sounds back (ctrl-shift-R)

**Comment by tposney** [comment]
In GitLab by @jdonley18 on Nov 25, 2020, 12:19

I'm self-hosting, so ctrl-shift-r didn't do anything noticeable.  I did
unmark the checkbox for custom sounds, saved the settings, then exited to
setup, then reloaded the world and I'm still getting no default dice roll.
I did find a workaround that I'm happy with for the moment, though, by
enabling the custom sounds and inserting the default dice roll sound to
everything through midi.  I'm mostly just posting so you know I'm getting
an error.

**Comment by tposney** [comment]
One thing I forgot to mention. If you use the combo card default roll sounds are disabled - the reason being is that the inbuilt dbd5e chat card is what causes the dice roll sounds. Since the merge card does not use the default roll chat card.

The merge card is the main reason that I added the custom sounds.

I think the hierarchy of sound generation is:
If merge card enabled sounds only come if midi-qol sounds are enabled
If no merge card, sounds enabled according to dice so nice settings. (I disabled midi-qol and with dice so nice enabled - no sounds). Midi-qol enabled, dice-so-nice disabled, no merge card defaults sounds come back.
